packages/navigator/src/browser/navigator-widget.tsx

```diff
@@ -101,11 +101,8 @@ export class FileNavigatorWidget extends FileTreeWidget {
     protected enableDndOnMainPanel(): void {
         const mainPanelNode = this.shell.mainPanel.node;
         this.addEventListener(mainPanelNode, 'drop', async ({ dataTransfer }) => {
-            const treeNode = dataTransfer && this.getTreeNodeFromData(dataTransfer) || undefined;
-
-            if (FileNode.is(treeNode)) {
-                this.commandService.executeCommand(CommonCommands.OPEN.id, treeNode.uri);
-            }
+            const treeNodes = dataTransfer && this.getSelectedTreeNodesFromData(dataTransfer) || [];
```

Comment:
`|| []`? looking at implementation of `getSelectedTreeNodesFromData` it cannot return `undefined`, please declare a return type explicitly

Reply:
This is for the case when dataTransfer value is null.